dwave/cloud/config/models.py

```diff
@@ -125,6 +127,42 @@ class LongPollingSchedule(BaseModel):
     pause: Optional[float] = 0.0
 
 
+class CacheConfig(BaseModel):
+    """Client cache configuration."""
+    # keeping it simple for now
+
+    #: cache enabled? inferred from `home`
+    enabled: bool = False
+
+    #: path to cache base directory; default to None if cache disabled, or
+    #: to `homebase.user_cache_dir` otherwise
+    #: Note: `home` sentinel values take precedence over `enabled`
+    home: Annotated[str | None, Field(default_factory=get_cache_dir)]
+
+    @model_validator(mode='after')
+    def process_home_sentinels(self):
+        if self.enabled and self.home is None:
+            self.home = 'default'
+
+        # expand home sentinels (override enabled)
+        if self.home is not None:
+            home = self.home.lower()
+            if home in ['off', 'disabled']:
+                self.enabled = False
+                self.home = None
+            elif home == 'default':
+                self.enabled = True
+                self.home = get_cache_dir()
```

Comment:
Should caching always be enabled when "home" is set _even_ if explicitly set to false?

Reply:
Caching is enabled only if a sentinel value `default` is used for `home`, so that a user can easily enable it from the env, even if it's explicitly disabled in the config file.